extension/popup.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Apiary Browser Extension</title>
+  <meta charset=utf8>
+  <style>
+    html {
+      width: 350px;
+      text-align: center;
+    }
+  </style>
+</head>
+
+<body>
+  <p>Apiary extension has been correctly loaded.</p>
```

Comment:
I'd change it to some more user-friendly message: `Apiary extension is ready for requests` or similar